pyproject.toml

```diff
@@ -14,22 +14,26 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
 
 [tool.poetry.dev-dependencies]
-black = "20.8b1"
+black = "21.9b0"
 pytest = "6.2.2"
 wheel = "0.35.1"
 mypy = "0.790"
-isort = "5.7.0"
+isort = "5.9.3"
 Django = "3.1.7"
-flake8 = "3.8.4"
+flake8 = "4.0.1"
 psycopg2 = "2.8.6"
+flake8-pyi = "^20.10.0"
 
 [tool.black]
 line-length = 88
 include = '\.pyi?$'
 
+[tool.pyright]
+useLibraryCodeForTypes = true
```

Comment:
I think we want this disabled, we really only want to be using type stubs for this project to make sure they're working

Reply:
otherwise looks good!

Reply:
Done!